docs/build.md

```diff
@@ -2,40 +2,84 @@
 
 Dependency | Version | Required           | Runtime
 ---        | ---     | ---                | ---
-Kokkos     | 3.1.00  | :heavy_check_mark: | :heavy_check_mark:
+[Kokkos](https://github.com/kokkos/kokkos)     | 3.1.00  | :heavy_check_mark: | :heavy_check_mark:
 MPI        | 2       |                    | :heavy_check_mark:
 CMake      | 3.12    | :heavy_check_mark: |
 
-# Installation
+# Build and installation
 
-## Installation using Spack
+## CMake
 
-ArborX can be installed using [Spack](https://github.com/spack/spack) package manager:
+Assuming that Kokkos is installed in `$KOKKOS_INSTALL_DIR`, configure ArborX
+```CMake
+ARGS=(
+    -D CMAKE_BUILD_TYPE=RELWITHDEBINFO
```

Comment:
Are you sure the spelling is ok.  I thought it was camelcase.
https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html

I would almost rather not mention it
https://github.com/arborx/ArborX/blob/52a3a64b713126d1ea97508a0cb3250d4a4d2ac7/CMakeLists.txt#L95-L101

Reply:
You are right, [latest](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html) CMake documentation spells it as `RelWithDebInfo`. I've always used it capitalized, though, I think they are backwards compatible.

The only reason I want to mention it here is that if users encounter an issue using ArborX, it would be easier for them to send us a backtrace and some debugging info.

Reply:
But my point was it is the default.

Reply:
Default for what? I don't care what the default is. My point: if inexperienced user grabs ArborX, builds it following the provided instructions, and encounteres problems, wouldn't it be better if he had some debug info enabled to speed up our help?

Reply:
If they omit `CMAKE_BUILD_TYPE` it will be set to `RelWithDebInfo` by the snippet of code I posted.

Reply:
Sure. Look, I don't have a strong opinion on this. I thought it made sense. I am OK with changing it.